### PR TITLE
Fix audio support on Windows7

### PR DESCRIPTION
**Pull request by kencooke**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 84912546646ce6806ad00016771b38abb44b3958 available at: https://gist.github.com/45ad2501da104b927fcbe8f9b2f3db15<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-8491254.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-8491254.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 372b6314485f4947c51fb85d0be327a850014881 available at: https://gist.github.com/45ad2501da104b927fcbe8f9b2f3db15<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-372b631.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-372b631.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: eb5b03d00836284102199c0711610af7484a67b8 available at: https://gist.github.com/45ad2501da104b927fcbe8f9b2f3db15<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-eb5b03d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-eb5b03d.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 541f7ebb05b12e8c1ced65cab3c65c65d8847828 available at: https://gist.github.com/45ad2501da104b927fcbe8f9b2f3db15<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-541f7eb.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-541f7eb.dmg

**Comment by kencooke** [comment]
This PR fixes audio for Windows 7 and earlier.

### Test Plan

1. Verify that audio devices are detected and audio works correctly, on Windows 7.
You may need to increase output buffers to avoid glitching.
In "\Program Files\High Fidelity - PR8721\audio you should only see qtaudio_windows.dll

2. Verify that audio devices are detected and audio works correctly, on Windows 8 or 10.
In "\Program Files\High Fidelity - PR8721\audio you should only see qtaudio_wasapi.dll and .pdb

**Comment by hifi-gustavo** [comment]
Build console output for commit: 40393bd8efdb368708167d4ef491cc83f2b0c541 available at: https://gist.github.com/45ad2501da104b927fcbe8f9b2f3db15<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-40393bd.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-40393bd.dmg

File: cmake/macros/PackageLibrariesForDeployment.cmake [hunk 1]
```diff
@@ -44,13 +44,25 @@ macro(PACKAGE_LIBRARIES_FOR_DEPLOYMENT)
...
-    )
+    if (DEPLOY_PACKAGE)
+      # copy qtaudio_wasapi.dll alongside qtaudio_windows.dll, and let the installer resolve
+      add_custom_command(
+        TARGET ${TARGET_NAME}
+        POST_BUILD
+        COMMAND if exist ${QTAUDIO_PATH}/qtaudio_windows.dll ( ${CMAKE_COMMAND} -E copy ${WASAPI_DLL_PATH}/qtaudio_wasapi.dll ${QTAUDIO_PATH} && ${CMAKE_COMMAND} -E copy ${WASAPI_DLL_PATH}/qtaudio_wasapi.pdb ${QTAUDIO_PATH} )
+        COMMAND if exist ${QTAUDIO_PATH}/qtaudio_windowsd.dll ( ${CMAKE_COMMAND} -E copy ${WASAPI_DLL_PATH}/qtaudio_wasapid.dll ${QTAUDIO_PATH} && ${CMAKE_COMMAND} -E copy ${WASAPI_DLL_PATH}/qtaudio_wasapid.pdb ${QTAUDIO_PATH} )
+      )
+    elseif (${CMAKE_SYSTEM_VERSION} VERSION_GREATER 6.2)
```
Comment (huffman) [inline cmake/macros/PackageLibrariesForDeployment.cmake]: Should this be > 6.1? I think 6.2 is Windows 8.

**Comment by huffman** [comment]
👍

File: cmake/macros/PackageLibrariesForDeployment.cmake (hunk 1 above)
Comment (kencooke) [inline cmake/macros/PackageLibrariesForDeployment.cmake]: It worked only because Windows 8 > 6.2.0000. I fixed it to be more clear.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6838dc0247cebc1c389a68c2be17601d0358a861 available at: https://gist.github.com/45ad2501da104b927fcbe8f9b2f3db15<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-6838dc0.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8721-6838dc0.dmg

**Pull request merged**